activesupport/lib/active_support/railtie.rb

```diff
@@ -85,11 +85,6 @@ class Railtie < Rails::Railtie # :nodoc:
     initializer "active_support.set_hash_digest_class" do |app|
       config.after_initialize do
         if app.config.active_support.use_sha1_digests
-          ActiveSupport::Deprecation.warn(<<-MSG.squish)
-            config.active_support.use_sha1_digests is deprecated and will
-            be removed from Rails 6.2. Use config.active_support.hash_digest_class
```

Comment:
I think we should say how people should use the `hash_digest_class` option in the deprecation message.